docs/source/sharing.rst

```diff
@@ -35,6 +35,10 @@ The sharing information of a content object can also be directly accessed by app
 .. literalinclude:: _json/sharing_folder_get.resp
    :language: http
 
+.. note::
+    Users and/or groups without a sharing entry can be found by appending the argument `search` to the query string. ie search=admin.
```

Comment:
@jaroel it would be nice to include a documentation example of the usage of the search term to test_documentation.py . I think it will be common enough use case to have it's own example.